operations/firmware/FAS_Recipes.md

```diff
@@ -192,13 +190,12 @@ Refer to [FAS Filters](FAS_Filters.md) for more information on the content used
 
 <a name="hpe-device-type-nodebmc-target--aka-bios"></a>
 
-#### Device Type: NodeBMC | Target: `System ROM` aka BIOS
-
-**IMPORTANT:**
-* If updating the System ROM of an NCN, the NTP and DNS server values will be lost and must be restored. For NCNs **other than ncn-m001** this can be done using the `/opt/cray/csm/scripts/node_management/set-bmc-ntp-dns.sh` script. Use the `-h` option to get a list of command line options required to restore the NTP and DNS values.
-See [Configure DNS and NTP on Each BMC](../../install/deploy_final_ncn.md#configure-dns-and-ntp-on-each-bmc)
-* Node should be powered on for System ROM update and will need to be rebooted to use the updated BIOS.
+### Device Type: NodeBMC | Target: `System ROM` aka BIOS
 
+>**IMPORTANT:**
+>* If updating the System ROM of an NCN, the NTP and DNS server values will be lost and must be restored. For NCNs **other than ncn-m001** this can be done using the `/opt/cray/csm/scripts/node_management/set-bmc-ntp-dns.sh` script. Use the `-h` option to get a list of command line options required to restore the NTP and DNS values.
```

Comment:
```suggestion
>* If updating the System ROM of an NCN, the NTP and DNS server values will be lost and must be restored. For NCNs **other than `ncn-m001`** this can be done using the `/opt/cray/csm/scripts/node_management/set-bmc-ntp-dns.sh` script. Use the `-h` option to get a list of command line options required to restore the NTP and DNS values.
```